### PR TITLE
Fix: Help menu links to the wrong user guide

### LINKED ISSUE #172: bug: Help menu leads to the wrong user guide
Currently, the help menu still leads to the original ab3 user guide.

### PR DESCRIPTION
**Pull request by delishad21**
Currently, the help button at the top of the help displays a menu that links back to the original ab3 user guide. Lets change the link so that it guides our users to the correct guide. 

Fixes #172

### CODE REVIEW

**Review by Zer0Legion — COMMENTED** [review]
Good catch! We definitely have to link back to our own user guide.

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL = "https://ay2324s2-cs2103t-t10-1.github.io/tp/UserGuide.html";
```
Comment (Pughal77) [inline src/main/java/seedu/address/ui/HelpWindow.java:18]: Nice you editted the link at the right place

**Review by Pughal77 — COMMENTED** [review]
Nice!

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (jayllo-c) [inline src/main/java/seedu/address/ui/HelpWindow.java:18]: Great find! Glad it now links to the correct user guide.

**Review by jayllo-c — COMMENTED** [review]
Good job!

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (danielleloh) [inline src/main/java/seedu/address/ui/HelpWindow.java:18]: good catch!

**Pull request merged**